apps/bot/commands/slash/get-answer-count.ts

```diff
@@ -0,0 +1,64 @@
+import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommand } from '../types.js'
+import {
+  getCorrectAnswersCount,
+  isUserProfilePublic,
+} from '../../db/actions/users.js'
+
+export const command: SlashCommand = {
+  data: new SlashCommandBuilder()
+    .setName('get-answer-count')
+    .setDescription('Get the number of your accepted answers')
+    .addUserOption((user) =>
+      user
+        .setName('user')
+        .setDescription('The user to get number of marked answers for')
+        .setRequired(false),
+    )
+    .setDMPermission(false),
+
+  async execute(interaction) {
+    // Get the user option
+    const userOption = interaction.options.getUser('user')
+
+    // Use the ID of the user option if it's provided, otherwise use the ID of the interaction user
+    const userId = userOption?.id || interaction.user.id
+    const userArgProvided = !!userOption
+    const count = await getCorrectAnswersCount(userId)
+    if (!count) {
+      await interaction.reply({
+        content: `It looks like ${
+          userArgProvided ? 'this user is' : 'you are'
+        } new to the forum! Start by answering some questions and you'll see your progress here.`,
```

Comment:
At the moment, if `userArgProvided` is truthy, the message is

> It looks like this user is new to the forum! Start by answering some questions and you'll see your progress here.

which sounds awkward if I already have had marked answers. Only add the "Start by..." sentence for when `userArgProvided` is falsy.